 wand2servo:
  ros__parameters:
    #all measures in cm

    #flap2servo
    hc1: 5.5  
    c1: 3.8
    b1: 14.0 ###
    D1_1: 17.0 ###
    a2: 2.0 
    b2: 129.0 
    c2: 2.2 

    #wand2flap
    L_rod: 200.7
    L_foil: 129.0
    OA: 2.82 #radius, to be updated!!! 
    Xoc: -200.7
    BC: 6.0
    CD: 1.9
    Yoc: -6.0
    #Yof: 1.0
    EF: 2.2
